apps/web/src/server/applications/case/representations/representation-details/representation-details.utilities.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @typedef {import('./application-representation-details.view-model.js').Representation} Representation
+ */
+
+/**
+ *
+ * @param {*} errors
+ * @returns {object}
+ */
+export const getFormattedErrorSummary = (errors) =>
+	Object.keys(errors).map((error) => ({
+		text: errors[error].msg,
+		href: `#${error}`
+	}));
+
+/**
+ * @param {Representation} representation
+ * @returns organisation name or firstname + lastname
+ */
+export const getOrgOrNameForRepresentation = ({ contacts }) => {
```

Comment:
you have the represented at this point why can't this just be (or simpler):

```
represented.organisationName ? represented.organisationName : represented.fullName,
```

All caller would need to do the contacts to represented but that would be a good seperation.

We have put effort into having the represented / representative why not use it.  

see - https://github.com/Planning-Inspectorate/back-office/pull/1128/files#diff-784408a94fb9cb6cad66f75dbdbbe207f282fa42c1f0a8817e960e4abedc9cd8R13

Reply:
as above